examples/generative_models/jtnn/jtnn/datautils.py

```diff
@@ -118,7 +121,10 @@ def __getitem__(self, idx):
         else:
             stereo_cand_graphs = []
             stereo_atom_x_enc = torch.zeros(0, atom_x_enc.shape[1])
-            stereo_bond_x_enc = torch.zeros(0, bond_x_enc.shape[1])
+            try:
+                stereo_bond_x_enc = torch.zeros(0, bond_x_enc.shape[1])
+            except:
+                stereo_bond_x_enc = bond_x_enc
```

Comment:
I assume the issue arises with a SMILES consisting of a single atom? What is the value of `bond_x_enc` in the two different cases?

Reply:
Just tried,

```python
import torch
a = torch.tensor([])
a.shape[1]
# IndexError: tuple index out of range
```